code/modules/research/designs.dm

```diff
@@ -583,11 +583,26 @@
 /datum/design/item/weapon/contactbeam
 	name = "C99 Supercollider Contact Beam"
 	id = "contactbeam"
-	req_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 5, TECH_POWER = 3)
-	materials = list(MATERIAL_STEEL = 5000, MATERIAL_GLASS = 1000, "uranium" = 500)
+	req_tech = list(TECH_COMBAT = 5, TECH_MATERIAL = 5, TECH_POWER = 3)
+	materials = list(MATERIAL_STEEL = 5000, MATERIAL_GLASS = 1000, "diamond" = 400)
```

Comment:
is diamond reasonably attainable? I put this here originally just taking the place (and costs) of the advanced energy gun, since people were making those

contact beam isn't meant to be especially rare if science wants to manufacture them

Reply:
Diamond happens to appear on most rounds, however raising the combat tech is just an unneeded, superfluous change. I'm not sure why it had to be made.

Reply:
@NanakoAC

Diamond became decently attainable now that the drill heads are fixed and can be used.